aries-site/src/pages/components/layer/index.mdx

```diff
@@ -0,0 +1,157 @@
+
+<StyleTable
+  data={[
+    {
+      label: 4,
+      property: 'pad',
+      value: 'medium',
+    },
+    {
+      label: 5,
+      property: 'gap',
+      value: 'medium',
+    },
+    {
+      label: 6,
+      property: 'round',
+      value: 'small',
+    },
+    {
+      label: 7,
+      property: 'elevation',
+      value: 'large',
+    },
+  ]}
+/>
+
+## Types of layers
+
+Different use cases and content warrant different layer types. Learn more about each type below.
+
+<LayerTypes />
+
+## Layer closing behavior
+
+### Mouse
+
+The closing behavior and anatomy of a layer will differ depending on if the layer is informational or actionable.
+
+#### Informational
+
+For informational layers that do not require any user actions, place a close button in the layer header.
+
+<Example plain>
+  <InformationalLayerClose />
+</Example>
+
+- The "X" button should close the layer.
+- The user can also click outside of the layer to close it.
+
+#### Actionable
+
+For actionable layers that require user actions, place a "Cancel" action alongside the other footer actions. In some cases, a label other than "Cancel" may be appropriate.
+
+Do not include an "X" button in the header to avoid confusion of how the user should close the layer.
+
+<Example plain>
+  <ActionableLayerClose />
+</Example>
+
+- The "Cancel" button should reject any changes and close the layer.
+- To prevent the user from losing progress due to accidentally misclicking outside of a layer, they should not be able to click outside of the layer to close it.
+
+<Notification
+  status="info"
+  message="The ordering and alignment of footer actions depends on the type of layer. See anatomy diagrams for specific layer types for details."
+  actions={[{ label: 'See layer types', href: '#types-of-layers' }]}
+  width="large"
+  margin={{ bottom: 'medium' }}
+/>
+
+### Keyboard
+
+| Key | Interaction                                                                                                                                                                                         |
+| --- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Esc | Close the layer and return focus to the element on the page the originally opened the layer. If the user has input any data into the layer, provide a double confirmation before closing the layer. |
+
+## Action label conventions
+
+If a layer contains footer actions, the primary action should be labeled using verb + noun format (e.g., "Apply filters").
+
+For any additional actions beyond the primary action, the noun may ommitted if you feel there will not be any ambiguity for the user with regards to what the action is referring to (e.g., "Reset" instead of "Reset filters").
+
+When in doubt, follow the verb + noun convention to ensure clarity.
+
+## Overflow behavior
```

Comment:
This may benefit from an interactive example.

We should also speak to where the scrollable region appears, as well as no sticky headers or sticky header only on scroll up. (part of last office hours discussion)